pyccel/commands/lambdify.py

```diff
@@ -0,0 +1,89 @@
+"""
+File describing commands associated with the lambdify function which converts a
+SymPy expression into a Pyccel-accelerated function.
+"""
+import sympy as sp
+from packaging import version
+
+from pyccel.epyccel import epyccel
+from pyccel.utilities.strings import random_string
+
+if version.parse(sp.__version__) >= version.parse('1.8'):
+    from sympy.printing.numpy import NumPyPrinter
+else:
+    from sympy.printing.pycode import NumPyPrinter
+
+def lambdify(expr : sp.Expr, args : 'dict[sp.Symbol, str]', result_type : str = None,
+             templates : 'dict[str,list[str]]' = None, **kwargs):
```

Comment:
I feel that we should make `result_type` and `templates` keyword-only arguments, as this will avoid many possible user errors.
```suggestion
def lambdify(expr : sp.Expr, args : 'dict[sp.Symbol, str]', *, result_type : str = None,
             templates : 'dict[str,list[str]]' = None, **kwargs):
```
What do you think?

Reply:
I agree. This should make it clearer. I will update this and the docs

Reply:
Done : 37a553b3b007690cf8831974ec155173ec8a61ba